src/tools/interop/idt/README.md

```diff
@@ -0,0 +1,564 @@
+environment.
+
+### Environment details
+
+1. This guide and the `idt` scripts expect **_one_** instance of `idt` on both
+   the admin computer and the target environment.
+1. The expected install location on the target environment is the home directory
+   of the user specified in `idt/scripts/vars.sh`. Default values are:
+    ```
+    PIHOST="kali-raspberry-pi"
+    PIUSER="kali"
+    ```
+1. Helper scripts may be used on admin computers that support `zsh` and `bash`
+   (Linux and macOS).
+1. Windows may be used as the admin computer via tools like `PowerShell`,
+   `MobaXterm` and `FileZilla`.
+1. This setup is intended to work with the admin computer and RPi connected to
+   the same Wi-Fi network, which is also the Wi-Fi network used for testing.
+1. Corporate networks are not expected to be used as test networks.
+
+### Prepare the RPi
+
+1. A >= 128 GB SD card is recommended.
+1. Flash the RPi SD with the latest
+   `Kali Linux RaspberryPi 2, 3, 4 and 400 (img.xz)` image (32 bit).
+    - Use [Raspberry Pi Imager](https://github.com/raspberrypi/rpi-imager) to
+      [download the image and prepare the SD](https://www.kali.org/docs/arm/using-rpi-imager-to-write-raspberry-pi-images/).
+1. Plug the SD into the RPi.
+1. Connect the RPi to your network via **ethernet.** [TODO] provide alternative
+1. Boot the RPi.
+
+### Configure admin computer and push to the RPi
+
+#### Linux and macOS admin computers
+
+1. On your admin computer, source the `alias` script from the parent directory
+   of `idt` to get `idt` commands in your current shell.
+    ```bash
+     source idt/scripts/alias.sh
+    ```
+    - To avoid having to repeat this step for each session, optionally configure
+      automatic aliases permanently.
+    - **_NOTE:_** Once run, `idt` commands will be globally and automatically
+      available. If you need to remove the installation, edit the `.rc` files
+      mentioned in `setup_shell`.
+    ```bash
+    source idt/scripts/setup_shell.sh
+    ```
+1. Send `idt` to the RPi:
+    ```
+    idt_push
+    ```
+1. `ssh` to the PRi (the default password for Kali is `kali`):
+    - **_NOTE:_** You may need to wait a few minutes after boot for the `ssh`
+      server to be available on the RPi. Retry if needed!
+    ```
+    idt_connect
+    ```
+
+#### Windows admin computers
+
+1. Open `PowerShell`, cd to the directory containing `idt` and send `idt` to the
+   RPi:
+    ```
+    scp -r ./idt/* kali@kali:/home/kali/idt
+    ```
+1. `ssh` to the RPi, e.g. with `MobaXterm` (the default password for Kali is
+   `kali`)
+    - **_NOTE:_** You may need to wait a few minutes after boot for the `ssh`
+      server to be available on the RPi. Retry if needed!
+    - Use `kali@kali-raspberry-pi` or `kali@$ip` where `$ip` is the RPi's IP
+      found in your router admin panel.
+
+### Configure the RPi
+
+1. Once in an ssh session, set a strong password.
+    ```
+    sudo passwd kali
```

Comment:
Setting up some SSH keys between the host and target would make connect easier.

Reply:
https://github.com/project-chip/connectedhomeip/pull/29174#discussion_r1330805987